src/conf.c

```diff
@@ -230,7 +230,7 @@ conf_init(conf_t *conf)
 	conf->buffer_size = 5120;
 	conf->max_speed = 0;
 	conf->verbose = 1;
-	conf->alternate_output = 0;
+	/* conf->alternate_output = 0; */
```

Comment:
The line should be removed, no reason to keep it.

Reply:
ok, delete this line.